sql_generators/serp_events/templates/desktop_query.sql

```diff
@@ -193,64 +188,25 @@ SELECT
   sample_id,
   experiments,
   is_shopping_page,
+  is_private,
```

Comment:
Does this mean we have info about PBM now?

Reply:
yes, this was [added](https://bugzilla.mozilla.org/show_bug.cgi?id=1851495) in 120.

Reply:
Nice!

Reply:
@dzeber Are there any metrics that we should not be tracking in PBM, or are we tracking everything just like in non-PBM mode?

Reply:
Not sure, I haven't followed the discussion on this update or dug into the data. I saw this flag was added to the event so I wanted to capture it in this table.